tests/test_switch/test_switchfeature.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+
+from mdssdk.switch import UnsupportedFeature, UnsupportedConfig
+from tests.test_switch.switch_vars import *
+
+log = logging.getLogger(__name__)
+
+class TestSwitchFeature(unittest.TestCase):
+    
+    def setUp(self) -> None:
+        self.switch = sw
+        log.info(sw.version)
+        log.info(sw.ipaddr)
+        self.name = 'analytics'
```

Comment:
pick one feature already enabled and pick one feature already disabled try do the tests on those..dont hardcode 'analytics', and while picking the feature dont pick telent/ssh/nxapi